spec/requests/messages_spec.rb

```diff
@@ -45,33 +45,64 @@
 
   describe 'PATCH /message/:id' do
     let(:previous_text) { 'Previous text' }
-    let(:message) { create(:message, creator_id: user.id, text: previous_text, request: request) }
     let(:new_attrs) { { text: 'Grab your coat and get your hat' } }
-    subject { -> { patch organization_message_url(message.organization, message, as: user), params: { message: new_attrs } } }
 
-    it { should change { message.reload && message.text }.from(previous_text).to('Grab your coat and get your hat') }
+    context 'with message not from the organization prevents updating ' do
+      let(:message) { create(:message, creator_id: user.id, text: previous_text) }
 
-    it 'shows success notification' do
-      subject.call
-      expect(flash[:success]).not_to be_empty
-    end
+      subject { -> { patch organization_message_url(organization, message, as: user), params: { message: new_attrs } } }
 
-    it 'redirects to the conversation link the message belongs to' do
-      subject.call
-      expect(response).to redirect_to(message.chat_message_link)
+      it 'renders a notfound' do
```

Comment:
```suggestion
      it 'renders not found' do
```

shared example?

Reply:
Because the shared spec is only for unpermitted users. Here the not found is caused by a message not belonging to the org

Reply:
there is still a type here where there is no space between not and found